amplipi/streams.py

```diff
@@ -1719,4 +1714,4 @@ def stream_types_available() -> List[str]:
     stypes.append(FMRadio)
   # the below line is not type checked because mypy isn't smart enough to see this is a relatively
   # constrained set of types, and instead evaluates this as a `list[type]`
-  return [s.stream_type for s in stypes] # type: ignore
+  return [s.stream_type for s in stypes] # type: ignore
```

Comment:
Weird that this doesn't have a newline, editorconfig?

Reply:
Fix the neline and I think this PR makes sense.

Reply:
Removed the newline